tests/Doctrine/Tests/DBAL/Platforms/AbstractSQLServerPlatformTestCase.php

```diff
@@ -579,14 +579,83 @@ public function testGetCreateSchemaSQL()
         self::assertEquals('CREATE SCHEMA ' . $schemaName, $sql);
     }
 
+    /**
+     * @group DBAL-42
```

Comment:
Where does the 42 come from?

Reply:
All tests relatet to column comments in AbstractPlatformTestCase.php are annotated with this group. I thought it would be important to keep it grouped. However, if thats not true I will remove it.